deployment/aks-periscope.yaml

```diff
@@ -132,6 +134,14 @@ metadata:
 data:
   DIAGNOSTIC_NODELOGS_LIST: /var/log/azure/cluster-provision.log /var/log/cloud-init.log
 ---
+apiVersion: v1 
+kind: ConfigMap 
+metadata:
+  name: clustertype-config
+  namespace: aks-periscope
+data:
+  CLUSTER_TYPE: "managedCluster" # <custom flag>
```

Comment:
A request: would it be possible to use a generic name here like `FEATURE_FLAGS`? Specifically because in PR #48, based on feedback from @Tatsinnit, we are looking to reuse your flag for adding in 2 new collectors (SMI and OSM) that are not based on cluster type.

Ref: https://github.com/Azure/aks-periscope/pull/48/files#diff-cdb15eaf022fc76984176a66a52183b8c3cb0710176cc9349519c5d43fb1d2ffR49-R58

Reply:
💡 Let's do this change in your PR, I think `ARC` folks have almost completed this, so as it looks like only thing you need is renaming this flag and we can do that as part of the `OSM x SMI` workitem.

Reply:
Sounds good to me. We can make that change as a fast follow once this gets merged

cc: @shalier